pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
-opts = -s
+addopts = -s
```

Comment:
whats the difference betwen` opts` and `addopts`?

Reply:
Pragmatically, `addopts` is documented and appears to work, while (at least in my configuration) `opts` generates warnings we may have just been ignoring.